lib/Red/Driver/SQLite.pm6

```diff
@@ -156,7 +156,7 @@ multi method default-type-for(Red::Column $
 multi method default-type-for($ --> Str:D) is default {"varchar(255)"}
 
 
-multi method map-exception(X::DBDish::DBError $x where { .?code == 19 and .native-message.starts-with: "UNIQUE constraint failed:" }) {
+multi method map-exception(X::DBDish::DBError $x where { (.?code == 19 or .?code == 1555) and .native-message.starts-with: "UNIQUE constraint failed:" }) {
```

Comment:
Do you know why the error code has changed? Different versions with different errors? Maybe we need a better way to map errors in the future.

Reply:
I really have no idea. I've left the former one, just on case it's still used and not deprecated.